plugins/modules/k8s_scale.py

```diff
@@ -277,14 +287,7 @@ def _continue_or_exit(warn):
                     result = resource.patch(existing.to_dict()).to_dict()
                 else:
                     result = scale(
```

Comment:
Wrap this scale() call in try/except looking for a CoreException and if it catches one call module.fail_json

Reply:
Done!